src/models/UserProfile.js

```diff
@@ -0,0 +1,51 @@
+// import { DataTypes } from 'sequelize';
+// import sequelize from '../database/config/db.js';
+// import User from './user.model.js';
+
+// const UserProfile = sequelize.define('user_profile_info', {
+//   id: {
+//     type: DataTypes.INTEGER,
+//     allowNull: false,
+//     primaryKey: true,
+//     autoIncrement: true,
+//   },
+//   placeofliving: {
+//     type: DataTypes.STRING(225),
+//     allowNull: true,
+//   },
+//   dob: {
+//     type: DataTypes.DATEONLY,
+//     allowNull: true,
+//   },
+//   lang: {
+//     type: DataTypes.STRING(225),
+//     allowNull: true,
+//   },
+//   currency: {
+//     type: DataTypes.STRING(50),
+//     allowNull: true,
+//   },
+//   gender: {
+//     type: DataTypes.STRING(50),
+//     allowNull: true,
+//   },
+//   userId: {
+//     type: DataTypes.UUID,
+//     allowNull: false,
+//     references: {
+//       tabel: 'users',
+//       model: 'User',
+//       key: 'id',
+//       onDelete: 'CASCADE',
+//       onUpdate: 'CASCADE',
+//     },
+//   },
+// });
+// User.hasOne(UserProfile, {
+//   foreignKey: 'userId',
+//   onDelete: 'CASCADE',
+//   onUpdate: 'CASCADE',
+// });
+// UserProfile.belongsTo(User, { foreignKey: 'userId' });
+
+// export default UserProfile;
```

Comment:
Do you expect this to be merged? If you prefer to have User profile as a separate model then discuss with your team and do so

Reply:
i made it, in its separete schema called users, which will discuss with my team to use it as they will add some items a user can buy. and about model, we will use the same table as as it says user's info